mbuild/tests/test_cassandramcf.py

```diff
@@ -0,0 +1,273 @@
+import pytest
+
+import mbuild as mb
+from mbuild.tests.base_test import BaseTest
+from mbuild.utils.io import has_foyer
+
+
+class TestCassandraMCF(BaseTest):
+
+    def test_not_parameterized(self, ethane):
+        with pytest.raises(ValueError,match=r'MCF writing not supported without'):
+            ethane.save(filename='ethane.mcf',angle_style='harmonic',dihedral_style='opls')
+
+    def test_invalid_structure(self,ethane):
+        with pytest.raises(ValueError,match=r'requires parmed structure'):
+            mb.formats.cassandramcf.write_mcf(ethane,'ethane.mcf',
+                    angle_style='harmonic',dihedral_style='opls')
+
+    @pytest.mark.skipif(not has_foyer, reason="Foyer package not installed")
```

Comment:
We could probably move this decorator all the way to the class. Formally, the first two tests don't require foyer, but it seems unlikely people would get to the point of writing an MCF without having foyer